cmd/server/sdn.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+
+	moovhttp "github.com/moov-io/base/http"
+	"github.com/moov-io/ofac"
+
+	"github.com/go-kit/kit/log"
+	"github.com/gorilla/mux"
+)
+
+// GET /sdn/:id/addresses
+// - get addresses for a given SDN
+
+// GET /sdn/:id/alternateNames
+// - get alternate names for a given SDN
+
+// GET /sdn/:id
+// - get SDN information
+
```

Comment:
Will you be adding a mock for SDN Comments?  The data set is very small, so address and AlternateIdentity seem critical path.

Reply:
I wasn't planning on it.